docs/tutorials/index.md

```diff
@@ -1,5 +1,5 @@
 # Tutorials
-
+git clean -xdi
```

Comment:
This command is can result in loss of data if not careful.  What problem is being solved by adding this?

Reply:
Depending on the OS it will clean `results` /`reports`/`logs` of previous runs. 
User must be careful, while selecting the option. So it is not recommended to include here.